tests/IResearch/IResearchQueryNGramMatchTest.cpp

```diff
@@ -35,7 +35,6 @@
 #include "VocBase/LogicalCollection.h"
 #include "store/mmap_directory.hpp"
 #include "utils/index_utils.hpp"
-#include "utils/string_utils.hpp"
```

Comment:
absl::Substitute is now used by this test. Should the corresponding header file be included here?

Reply:
:heavy_check_mark: